docs/victoriatraces/README.md

```diff
@@ -0,0 +1,112 @@
+VictoriaTraces is open source user-friendly database for distributed tracing data 
+from [VictoriaMetrics](https://github.com/VictoriaMetrics/VictoriaMetrics/).
+
+VictoriaTraces provides the following features:
+- It is resource-efficient and fast. It uses up to [**3.7x less RAM and up to 2.6x less CPU**](https://victoriametrics.com/blog/dev-note-distributed-tracing-with-victorialogs/) than other solutions such as Grafana Tempo.
+- VictoriaTraces' capacity and performance scales linearly with the available resources (CPU, RAM, disk IO, disk space).
+- It accepts trace spans in the popular [OpenTelemetry protocol](https://opentelemetry.io/docs/specs/otel/protocol/)(OTLP).
+- It provides [Jaeger Query Service JSON APIs](https://www.jaegertracing.io/docs/2.6/apis/#internal-http-json) 
+  to integrate with [Grafana](https://grafana.com/docs/grafana/latest/datasources/jaeger/) or [Jaeger Frontend](https://www.jaegertracing.io/docs/2.6/frontend-ui/).
+
+## How does it work
+
+VictoriaTraces is built on top of [VictoriaLogs](https://docs.victoriametrics.com/victorialogs/), which is a log database. 
+It receives trace spans in OTLP format, transforms them into structured logs, and provides [Jaeger Query Service JSON APIs](https://www.jaegertracing.io/docs/2.6/apis/#internal-http-json) for querying.
+
+For detailed data model and example, see: [Key Concepts](https://docs.victoriametrics.com/victoriatraces/keyConcepts).
+
+![How does VictoriaTraces work](how-does-it-work.webp)
+
+Building VictoriaTraces in this way enables it to scale easily and linearly with the available resources, like VictoriaLogs.
+
+## Quick Start
```

Comment:
I think Quick Start should focus on how one can set up VictoriaTraces and get some real results. I'd recommend showing how to set it up using Docker Compose, along with Grafana and a Jaeger data source preconfigured. To generate some sample traces, we can suggest using `jaegertracing/jaeger-tracegen` tool:

```
docker run --rm \
  --env=OTEL_EXPORTER_OTLP_ENDPOINT=http://victoriatraces:9428/insert/opentelemetry/ \
  --network=docker_default \
  jaegertracing/jaeger-tracegen:1.70.0 \
  -duration=1m
```

After running this, users should have some traces data in the storage. So we can navigate them to Grafana, select the Jaeger data source, and explore the generated traces.

@jiekun wdyt?

Reply:
I agree we should have a full demo. For vm and vlogs there're some docker compose in /deployment. I think we should just provide the link to there and avoid maintaining same things in different places.

Reply:
updated in 5a053d571